.github/workflows/build.yml

```
@@ -1,42 +1,11 @@
 name: Build
```

Maybe this could be re-named to _e2e-test_?

```suggestion
name: E2E-Test
```

... and the _image_ target could be separated as well?

---

yes, this makes sense. I would actually even remove the `image` target, since it's part of e2e build anyway